serverless/src/adminGetRetrievalsMetrics/handler.js

```diff
@@ -0,0 +1,87 @@
+import { getDbConnection } from '../util/database/getDbConnection'
+import { getApplicationConfig } from '../../../sharedUtils/config'
+import { parseError } from '../../../sharedUtils/parseError'
+
+/**
+ * Retrieve all the retrieval metrics for the authenticated user
+ * @param {Object} event Details about the HTTP request that it received
+ * @param {Object} context Methods and properties that provide information about the invocation, function, and execution environment
+ */
+const adminGetRetrievalsMetrics = async (event, context) => {
+  // https://stackoverflow.com/questions/49347210/why-aws-lambda-keeps-timing-out-when-using-knex-js
+  context.callbackWaitsForEmptyEventLoop = false
+  const { defaultResponseHeaders } = getApplicationConfig()
+
+  try {
+    const { queryStringParameters = {} } = event
+    const {
+      start_date: startDate,
+      end_date: endDate
+    } = queryStringParameters || {}
+
+    // Retrieve a connection to the database
+    const dbConnection = await getDbConnection()
+
+    // `jsonExtract` parses fields in `jsonb` columns
+    // https://knexjs.org/guide/query-builder.html#jsonextract
+    // Fetch metrics on `retrieval_collections`
+    const retrievalResponse = await dbConnection('retrieval_collections')
```

Comment:
This appears to be the most complex query we have, it would be a good idea to check the contents of this query in this test like we do in EDD (a good idea for all our queries really), to better ensure that it doesn't get broken later be mistake.

Same for the other query below

Reply:
Do you mean an assertion to check the exact payload of the SQL? such as `expect(query.sql).toEqual(<sql-string>)`?